attestation-service/docs/policy.md

```diff
@@ -81,4 +81,5 @@ We will introduce the format of policy by providing some examples to show the us
 
 1. The [default policy](../attestation-service/src/policy_engine/opa/default_policy.rego). This policy will check whether each entry in the [parsed claims](./parsed_claims.md) generated by the input evidence matches the reference value obtained from RVPS.
 2. An [SGX policy](../tests/coco-as/policy/example-1.rego). The client want to ensure the `mr_signer` and `mrenclave` are both expected value.
-3. A [TDX policy](../tests/coco-as/policy/example-2.rego). The client want to ensure the TDX module (reflected by `tdx.quote.body.mr_seam`), guest firmware (reflected by `tdx.quote.body.mr_td`), kernel (reflected by `tdx.ccel.kernel`) are all as expected.
\ No newline at end of file
+3. A [TDX policy](../tests/coco-as/policy/example-2.rego). The client want to ensure the TDX module (reflected by `tdx.quote.body.mr_seam`), guest firmware (reflected by `tdx.quote.body.mr_td`), kernel (reflected by `tdx.ccel.kernel`) are all as expected.
+4. A [IBM SE policy](../tests/coco-as/policy/example-3.rego). The client want to ensure the `se.version` and `se.tag` and `se.user_data` and `se.image_phkh` and `se.attestation_phkh` are both expected value.
```

Comment:
nit:
the `se.version` and `se.tag` and `se.user_data` and `se.image_phkh` and `se.attestation_phkh` are both expected value.  =>

the `se.version`, `se.tag`, `se.user_data`, `se.image_phkh` and `se.attestation_phkh` are all expected value.

Reply:
fixed.